### ISSUE TITLE
500 Internal Server Error on some Linux installs

### ISSUE THREAD

**Issue by villares** [berinhard/pyp5js#108]
On my installation I have this on the terminal after clicking on a sketch in the browser sketchbook page:

```
 :~$ pyp5js serve
 * Serving Flask app "pyp5js.http.web_app" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
127.0.0.1 - - [05/Nov/2019 00:14:10] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [05/Nov/2019 00:14:10] "GET /static/styles/basscss.min.css HTTP/1.1" 200 -
127.0.0.1 - - [05/Nov/2019 00:14:10] "GET /static/styles/custom.css HTTP/1.1" 200 -
Converting Python to P5.js...
Running command:
	 transcrypt -xp /usr/local/lib/python3.6/dist-packages/pyp5js -k -ks -b -m -n /home/villares/sketchbook-pyp5js/oioioi/target_sketch.py
[2019-11-05 00:14:14,485] ERROR in app: Exception on /sketch/oioioi/ [GET]
Traceback (most recent call last):
  File "/home/villares/.local/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/villares/.local/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/villares/.local/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/villares/.local/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/villares/.local/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/villares/.local/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python3.6/dist-packages/pyp5js/http/web_app.py", line 94, in render_sketch_view
    commands.transcrypt_sketch(sketch_name)
  File "/usr/local/lib/python3.6/dist-packages/pyp5js/commands.py", line 58, in transcrypt_sketch
    compile_sketch_js(sketch_files)
  File "/usr/local/lib/python3.6/dist-packages/pyp5js/compiler.py", line 69, in compile_sketch_js
    compiler.compile_sketch_js()
  File "/usr/local/lib/python3.6/dist-packages/pyp5js/compiler.py", line 16, in compile_sketch_js
    self.run_compiler()
  File "/usr/local/lib/python3.6/dist-packages/pyp5js/compiler.py", line 43, in run_compiler
    proc = subprocess.Popen(shlex.split(command))
  File "/usr/lib/python3.6/subprocess.py", line 729, in __init__
    restore_signals, start_new_session)
  File "/usr/lib/python3.6/subprocess.py", line 1364, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'transcrypt': 'transcrypt'
127.0.0.1 - - [05/Nov/2019 00:14:14] "GET /sketch/oioioi/ HTTP/1.1" 500 -
```

**Comment by berinhard** [comment]
This is odd because you shouldn't be able to install `pyp5js` without transcrypt since it's a dependency. What's the terminal output when you simply run:

```bash
$ transcrypt --help
```

**Comment by berinhard** [comment]
Maybe this error can also be fixed by #89.

**Comment by berinhard** [comment]
@villares this probably was fixed by #111 PR as well because it changes how the code executes `transcrypt` to a more system agnostic way. Also, we now have [CI tests](https://github.com/berinhard/pyp5js/actions/runs/87152825) to ensure the multiple environments (Windows, MacOs and Linux).

I'll release a new version of `pyp5js` next weed and it would be nice to know if this will be fixed.

**Comment by villares** [comment]
`pyp5js serve` worked recently on my Xubuntu, so I guess we should close this!
